### DISCUSSION TITLE
Unrecognized relative link when there is no link

### DISCUSSION THREAD

**Discussion by rajdevworks** [squidfunk/mkdocs-material#7139] [General]
I have following markdown content and it is a log snippet from a file. It has same/similar syntax of handling links in mkdocs.
It is throwing multiple INFO notices while serving the docs. How can I avoid getting these notices?

`>2018-03-15T05:36:13.035843+00:00 something some[6402]: dispose dev[serial](F02)...<br>`

Error while serving mkdocs
`INFO    -  Doc file 'filename.md' contains an unrecognized relative link 'F02', it was left as is.
`

**Comment by alexvoss** [comment]
`[]()` is the Markdown syntax for a link. What you have is not really Markdown but some other content that happens to look like a link to the Markdown parser. You need to either indent the lines with at least four spaces or a tab or surround such content with backticks:

    ```
    >2018-03-15T05:36:13.035843+00:00 something some[6402]: dispose dev[serial](F02)...<br>
    ```
